Source/carthage/Update.swift

```diff
@@ -65,13 +68,15 @@ public struct UpdateCommand: CommandProtocol {
 			let buildDescription = "skip the building of dependencies after updating\n(ignored if --no-checkout option is present)"
 
 			let dependenciesUsage = "the dependency names to update, checkout and build"
+            let netrcOption = Option(key: "use-netrc", defaultValue: false, usage: "use authentication credentials from ~/.netrc file when downloading binary only frameworks")
```

Comment:
Hey @tmspzz 👋 

I added the `use-netrc` option according to your suggestion, however I am not sure if `update` command is the only one where we shall add the option 🤔

Thoughts?